server/cmwell-cons/src/main/scala/GridSubDiv.scala

```diff
@@ -207,7 +211,7 @@ case class GridSubDiv(user: String,
             listenAddress = host,
             masterNodes = esMasters,
             sName = s"${ResourceBuilder.getIndexedName("start", i)}.sh",
-            index = i,
+            index = i + 1,
```

Comment:
1. the last one will clash with the master node. 
2. for Grid the master is the first one